src/qt/addresstablemodel.h

```diff
@@ -81,10 +81,10 @@ class AddressTableModel : public QAbstractTableModel
     OutputType GetDefaultAddressType() const;
 
 private:
-    WalletModel *walletModel;
-    AddressTablePriv *priv;
+    WalletModel *walletModel = nullptr;
```

Comment:
`walletModel` and `priv` are initialized in constructor:
https://github.com/bitcoin/bitcoin/blob/727175a08dffcb5f18f8faac562ab54f89e5f36e/src/qt/addresstablemodel.cpp#L162

Reply:
@promag See the pull request description – it was only `editStatus` and `autoCompleter` that were previously neither initialized where defined nor in constructor. The other ones:
> Also; initialize non-static class members where they are defined in accordance with developer notes.

Reply:
`walletModel` is initialized in constructor, why this? Maybe change here to
```cpp
WalletModel* const walletModel;
```
so it only compiles if initialized in constructor.